client/client-multi/client-c/tools/kaa_encryption/rsa_key_gen/kaa_rsa_key_gen.c

```diff
@@ -105,6 +103,7 @@ int rsa_genkey(mbedtls_pk_context *pk)
 void store_key(FILE *fd, const char *prefix, size_t prefix_size,
                       uint8_t *key, size_t length)
```

Comment:
Let's use `snprintf` instead of `sprintf` and add a check for potential overflow. Also, there is no need for such a large buffer in this particular case